docs/integrations/data-ingestion/clickpipes/postgres/lifecycle.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_label: 'Lifecycle of a Postgres ClickPipe'
+description: 'Various pipe statuses and their meanings'
+slug: /integrations/clickpipes/postgres/lifecycle
+title: 'Lifecycle of a Postgres ClickPipe'
+---
+
+# Lifecycle of a Postgres ClickPipe {#lifecycle}
+
+This is a document on the various phases of a Postgres ClickPipe, the different statuses it can have, and what they mean.
+
+## Provisioning {#provisioning}
+
+When you click on the Create ClickPipe button, the ClickPipe is created in a `Provisioning` state. The provisioning process is where we spin up the underlying infrastructure to run ClickPipes for the service, along with registering some initial metadata for the pipe. Since compute for ClickPipes within a service is shared, your second ClickPipe will be created much faster than the first one -- as the infrastructure is already in place.
+
+## Setup {#setup}
+
+After a pipe is provisioned, it enters the `Setup` state. This state is where we create the destination ClickHouse tables. We also obtain and record the table definitions of your source tables here.
+
+## Snapshot {#snapshot}
+
+Once setup is complete, we enter the `Snapshot` state. `Snapshot`, `Initial Snapshot` and `Initial Load` (more common) are interchangeable terms. In this state, we take a snapshot of the source database tables and load them into ClickHouse. This **does not use logical replication**. For more information on initial load, see the [parallel initial load documentation](./parallel_initial_load). The pipe will also enter the `Snapshot` state when a resync is triggered or when new tables are added to an existing pipe.
```

Comment:
we don't use logical replication, but may be important to note we create slot in setup & thus slot growth / retention periods need to account for this waiting around during snapshot